services/save_poll_rating/index.js

```diff
@@ -0,0 +1,88 @@
+const Assert = require('assert-plus')
+const { fetchProp } = require('../../lib/utils')
+const { NotFoundError } = require('../../lib/errors')
+
+// TODO: Consider renaming SavePollRating to DenormalizePollRating
+//
+class SavePollRating {
+  static async toEntities(args, ctx, opts = {}) {
+    Assert.object(args, 'args')
+    const poll_rating = fetchProp(args, 'rating')
+
+    Assert.object(ctx, 'ctx')
+    const seneca = fetchProp(ctx, 'seneca')
+
+    const ent_infos = getEntityInfoForSave(args, ctx, opts) 
+
+    for (const ent_info of ent_infos) {
+      const ent_name = fetchProp(ent_info, 'ent_name')
+      const ent_id = fetchProp(ent_info, 'ent_id')
+      const field = fetchProp(ent_info, 'field')
+
+      const ent = await seneca.make(ent_name)
+        .load$(ent_id)
+
+      if (!ent) {
+        throw new NotFoundError(ent_name)
+      }
+
+      await ent
+        .data$({ [field]: poll_rating })
+        .save$()
+    }
+  }
+}
+
+
+function getEntityInfoForSave(args, _ctx, opts) {
+  Assert.object(opts, 'opts')
```

Comment:
Should we, perhaps, implement validation of the plugin options, passed to the plugin by the client? Such validation would be implemented in `index.js`, and make sure client passes supported well-formed options.